src/writer.rs

```diff
@@ -3,36 +3,20 @@ use rust_xlsxwriter::{ColNum, RowNum, Worksheet};
 
 use crate::format::{self, ExcelFormat};
 
-/// Worksheet handler for writing string values.
```

Comment:
Killing docstrings in this file, as we're not going to directly call this function from Python anyway, and we have examples on the `write` method already. At this point, it just becomes a noise